src/pages/guide/language/function.md

```diff
@@ -91,16 +93,6 @@ let drawCircle = (~radius as r, ~color as c) => {
 drawCircle(~radius=10, ~color="red");
 ```
 
-For the common case of `radius::radius` (where the label is the same as the local variable name), we have the syntax shorthand `::x`:
-
-```reason
-let drawCircle = (~radius, ~color) => {
-  setColor(color);
-  startAt(radius, radius);
-  ...
-};
-```
-
```

Comment:
I believe this section is no longer applicable.

Reply:
This is still applicable but the wording needs to change a bit. `(~radius)` is shorthand for `(~radius as radius)`

Reply:
Okiedoke, I can take a crack at it once I grasp the distinction. Can you clarify the difference conceptually between these:

Reason: `(~radius) =>` is shorthand for `(~radius as radius) =>`
JS: `({ radius }) =>` is not really shorthand for `({ radius: radius }) =>` because it would be conceptually extraneous/redundant

Ignore the fact that passing an object and naming an arg are not the same thing, just referring to the concept of aliasing. As an outsider, the distinction between the two feels extraneous, like in that JS example?

Reply:
Both are just labeled punning for their more verbose counterpart, if that's what you mean?

Reply:
Gotcha. Mostly I feel this is an unnecessary distinction, and one that actually confuses more than helps--like it would confuse if I was teaching JS and said `({ radius }) =>` was short for `({ radius: radius }) =>`. Obv your call, just feedback.

Reply:
I'll try to craft an explanation later today, or you're welcome to patch on top. 👍

Reply:
K, removed, thanks!